### PR TITLE
Feature/2 populate measurement (Issue #2)

### PR DESCRIPTION
**Pull request by BarisSari**
Supports #2 

I reformatted all populate methods and added the populate_measurement method.

### CODE REVIEW

**Comment by IanMayo** [comment]
You've done a great job here @BarisSari

File: pepys_import/core/store/data_store.py [hunk 1]
```diff
@@ -705,6 +705,10 @@ def get_sensors_by_platform_type(self, platform_type):
             .all()
         )
 
+    def get_states(self):
+        # get list of all states in the DB
+        return self.session.query(self.db_classes.State).all()
```
Comment (IanMayo) [inline pepys_import/core/store/data_store.py]: I had _no idea_ how to do this :-)

Comment (BarisSari) [reply pepys_import/core/store/data_store.py]: Really? I wish I had implemented it before :)

Comment (IanMayo) [reply pepys_import/core/store/data_store.py]: I know _practically nothing_ about SQLAlchemy.

In recent weeks I've learned _a tiny bit more than that_ about Python.

**Pull request merged**